### DISCUSSION TITLE
Browser is always warning me to not open r2cloud IP

### DISCUSSION THREAD

**Discussion by ea5wa** [dernasherbrezon/r2cloud#111] [Q&A]
How can I tell to my browser that I trust in r2cloud?

Every time I open my r2cloud local IP, everytime have a warning asking me if I'm sure, because is not a secure website....

I need some help

73 de EA5WA Juan Carlos

**Comment by dernasherbrezon** [comment]
You can try adding exception for this site into the browser's cache. If it is still failing, then you might remove old certificate from the SSL store and re-import it again. 

What browser are you using?

**Reply by ea5wa** [reply]
It happens in every browser and both on Android mobile phone and windows computer

**Reply by dernasherbrezon** [reply]
Normally self signed certificate can be added to the trust store only once on the first visit. Here is the example for Safari:

![screen-shot-2020-04-18-at-11 27 32-pm](https://user-images.githubusercontent.com/1614424/124954637-364b4a80-e00e-11eb-8d5d-7f5a030ad245.png)

Can you try opening Rpi by the hostname, not IP. I.e. https://raspberrypi.local

**Comment by ea5wa** [comment]
The problem is due to the certificate..... I don't know why... 
![image](https://user-images.githubusercontent.com/54686689/124970829-b54d7c80-e028-11eb-897a-3737930417e7.png)

**Reply by dernasherbrezon** [reply]
That's because certificate is self-signed.  .local domains cannot be signed by trusted certificate authorities. Hence always self-signed.
